src/PAModel/PAConvert/ErrorCode.cs

```diff
@@ -141,16 +141,6 @@ public static void ParseError(this ErrorContainer errors, SourceLocation span, s
             errors.AddError(ErrorCode.ParseError, span, $"Parse error: {message}");
         }
 
-        public static void ValidationError(this ErrorContainer errors, string message)
```

Comment:
Alternatively, rather than entirely removing this you could consider just outputting these as a Warning...